Library/Tests/ViewModels/LiveStreamContainerViewModelTests.swift

```diff
@@ -173,7 +173,7 @@ internal final class LiveStreamContainerViewModelTests: TestCase {
 
     withEnvironment(liveStreamService: MockLiveStreamService(fetchEventResult: Result(nonLiveStreamEvent))) {
       self.vm.inputs.configureWith(project: project,
-                                   liveStreamEvent: nonLiveStreamEvent,
+                                   liveStreamEvent: liveStreamEvent,
```

Comment:
oof, good catch, but not sure i understand. would it be possible to change the name of this test and add a lil comment to the test to describe exactly what this test is supposed to capture?